Week01/Problem03/ademissie_03.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 13 14:23:54 2018
+
+@author: ademissie6
+"""
+
+import math
+
+number =600851475143
+ 
+def isPrime(factor):
+    for pfactors in range(2,int(math.sqrt(factor))+1):
+        if factor % pfactors == 0:
+            return False
+ 
+    return True
+
+for pfactors in range(2, number + 1):
+      if isPrime(pfactors) and number % pfactors == 0:
+         
```

Comment:
looks like you're missing a line here, when I run this code it throws an error and you seem to be missing stuff